Rakefile

```diff
@@ -21,3 +24,16 @@ end
 def run(*cmd)
   sh(cmd.join(' '))
 end
+
+RDoc::Task.new do |rdoc|
+  rdoc.main = "README.md"
+  rdoc.rdoc_files.include(%w{README.md LICENSE CONTRIBUTING.md lib ext})
+end
```

Comment:
Will this task run if you do a `gem build` on the gemspec? Is it not possible to add this to the gemspec itself?

Reply:
@v0dro I didn't get the question. Why would we need `rdoc` to run when using `gem build`.?

AFAIK, RubyDoc.info automatically generates the docs when I push the gem.

Reply:
So the above rake commands adds rdoc files which are then read by rubygems.org to generate YARD docs. If its not done during gem build and if you forget to run above rake task, the rdoc files will not get added to the gemspec and rubygems wont be able to detect them.

Reply:
@v0dro, I will follow Victor's advise about using `rubygems-tasks` library.